python-sdk-sample/src/main.py

```diff
@@ -8,4 +8,123 @@
 Copyright Contributors to the Zowe Project.
 """
 
-print("Hello world!")
+from zowe.core_for_zowe_sdk import SdkApi
+
+
+class SampleSdk(SdkApi):
+    """
+    Class used to represent the sample API.
+
+    ...
+
+    Attributes
+    ----------
+    connection
+        Connection object
+    """
+
+    def __init__(self, connection):
+        """
+        Construct a sample SDK object.
+
+        Parameters
+        ----------
+        connection
+            The connection object
+        """
+        super().__init__(connection, "https://jsonplaceholder.typicode.com", logger_name=__name__)
+        self.request_endpoint = self.default_service_url
```

Comment:
For the sample we don't want to require loading a z/OSMF profile, and should be able to pass connection info inline:
```suggestion
        super().__init__({"host": "https://jsonplaceholder.typicode.com", "user": "fake", "password": "fake"}, "/", logger_name=__name__)
```

The user and password are temporarily required until we enhance the Session class to support AUTH_TYPE_NONE (https://github.com/zowe/zowe-client-python-sdk/issues/291).

Reply:
@zFernand0 @traeok Do you think we should add a "sample" profile type to the Python SDK Sample to demonstrate loading profiles? I think the CLI sample plugin has something similar - a sample plugin that stores some silly properties like T-shirt size.

Reply:
I think a sample profile type would be a good example for this SDK, as long as it doesn't increase the scope too much 😋

Reply:
I don't think I can add a "sample" profile type here as it requires update on `zowe.config.json` which is not contained in this repo.

Reply:
We could add a `zowe.config.json` file to this repo but I think it could be done as a future enhancement 🙂